v4l-dvb/usb.mk

```diff
@@ -167,7 +167,7 @@ define KernelPackage/dvb-usb-dvbsky
   V4L_KCONFIG := CONFIG_DVB_USB_DVBSKY
   FILES := $(PKG_BUILD_DIR)/v4l/dvb-usb-dvbsky.ko
   AUTOLOAD := $(call AutoProbe,dvb-usb-dvbsky)
-  $(call AddDepends/dvb-usb-v2,+kmod-dvb-si2168 +kmod-dvb-m88ds3103)
+  $(call AddDepends/dvb-usb-v2,+kmod-dvb-si2168 +kmod-media-tuner-si2157 +kmod-dvb-m88ds3103 +kmod-dvb-sp2 +kmod-dvb-ts2020)
```

Comment:
Please drop this change. These modules are not necessary to load 'dvb-usb-dvbsky' kernel driver, they are only used for Your particular device. What I want is to support only minimal dependencies for each module, if we leave it like this, if there is some kind of pcie dvb card using at least one of those modules then either the package created for that device will have conflicting files or it'll have dependency on 'kmod-dvb-usb-dvbsky' which will pull unnecessary usb dependencies.
To solve this problem in my local tree (got whole lot other uncommitted changes, few of them overlap with Yours) I created dummy packages for single or multiple devices, for example:
````
define Package/generic-dib0700
  SECTION := dvb-devices
  CATEGORY := DVB devices
  SUBMENU := Terrestrial (DVB-T/T2)
  TITLE := Dibcom 0700 based
  DEPENDS := +dib0700-firmware +kmod-dvb-dib0070 +kmod-dvb-dib0090 +kmod-dvb-dib3000mc +kmod-dvb-dib7000m +kmod-dvb-dib7000p +kmod-dvb-usb-dib0700
endef

define Package/generic-dib0700/description
 Supported:
 Medion CTX1921 (V2.1.2) USB DVB-T dongle.
endef

define Package/generic-dib0700/install
	$(INSTALL_DIR) $(1)/etc
endef

$(eval $(call BuildPackage,generic-dib0700))
````
This dummy package will pull all necessary firmware/kmods for the device mentioned in the description. That way we won't have to bother about conflicts or unnecessary modules installed.

What are Your thoughts on this, are You fine with me pushing first my local changes and then You adjusting to them, or the other way around?

Reply:
Thanks for the explanation, I got it and agreed. I think it could be enough to warn unexperienced users about how it works. To maintain a large list of dummy packages for every devices out there is a huge overhead what no one wishes. Pls see my new PR.

Reply:
The dummy packages are only for convenience and they are not such a big burden, especially since lots of usb devices are clones, so adding them to description won't be difficult for average user. What needs to be done is to explain for the average user how to find necessary modules for her/his card and Your PR about readme partially explains it, thanks for that. Still the readme is severely lacking, I'll update it as the time goes.